PROVISIONING-METAL.md

```diff
@@ -194,6 +199,18 @@ to = "10.10.10.0/24"
 from = "192.168.14.5"
 via = "192.168.14.25"
 
+# Interfaces may be configured using their MAC address rather than the interface name.
+# The MAC address must be quoted.
+["0e:b3:69:44:b6:33"]
+dhcp4 = true
+
+["3e:03:69:49:e6:31".static4]
```

Comment:
So you can't refer to this interface in a bonding or vlan configuration because it has no name?

Reply:
Confirmed: wicked doesn't support this.